xgcm/grid.py

```diff
@@ -74,6 +73,7 @@ def __init__(self, ds, axis_name, periodic=True, default_shifts={},
         self._name = axis_name
         self._periodic = periodic
         self._wrap = wrap
+        self._raw_out = raw_out
```

Comment:
What is this attribute?!?

Reply:
This is basically a switch to omit the call to `_wrap_and_replace_coords` at the end. This way I can use all the logic from `xgcm.Axis` to do the interpolation (as mentioned above). I think this is much better then keeping a branched out interp function in the autogenerate module.